scylla/src/transport/iterator.rs

```diff
@@ -888,16 +930,14 @@ impl LegacyRowIterator {
         //   to the channel (the PageSendAttemptedProof helps enforce this)
         // - That future is polled in a tokio::task which isn't going to be
         //   cancelled
-        let pages_received = receiver.recv().await.unwrap()?;
-        let rows = pages_received
-            .rows
-            .into_legacy_rows(PagingStateResponse::NoMorePages)?;
+        let page_received = receiver.recv().await.unwrap()?;
```

Comment:
Any idea why this variable was previously called `pages_received` (plural)?

Reply:
Unfortunately, no.